core/src/sql/access_type.rs

```diff
@@ -260,23 +256,20 @@ pub struct JwtAccessVerifyJwks {
 	pub url: String,
 }
 
-#[revisioned(revision = 2)]
+#[revisioned(revision = 1)]
```

Comment:
This is a breaking change. We are essentially rolling back #4361. Is there a way to do this cleaner without keeping an unused field in the structure for the whole 2.X release?

Reply:
Another softer breaking change is obviously the fact that the `DEFINE ACCESS` syntax has changed slightly.

Reply:
That seems very reasonable. I will make this change for now :+1: 

My goal was to avoid introducing unnecessary disturbance if possible, even if we are still in alpha. I think this is good if we want to encourage people to keep using alpha, which is extremely valuable.

Reply:
Oh wait, what will happen to access methods that did have this field? :thinking: 
I think revisioning this properly may introduce unexpected behavior such as the `AUTHENTICATE` clause not running for those access methods, which could lead to security issues if users they updating SurrealDB up to a stable release without noticing the change. I think in this case it may be best to have a breaking change so that users notice that this change has taken place and ensure that the `AUTHENTICATE` clause is migrated.

Reply:
It is probably still best to make it a version 3 then.
Revision requires a convert_fn function on fields which no longer exists which is called whenever a version with that field is serialized.

You can return an error from this function so you could use that to make sure the an error happens when such a field is no longer supported.

Reply:
Oh, nice! Thank you for the suggestion. I will do that then :+1:

Reply:
Done in [10c8721](https://github.com/surrealdb/surrealdb/pull/4385/commits/10c8721114347f355de7c97713984de9680f1946), [527c98c](https://github.com/surrealdb/surrealdb/pull/4385/commits/527c98c8e7d60a30574cd014602495672d5aa91c) and [089b7b5](https://github.com/surrealdb/surrealdb/pull/4385/commits/089b7b53cfad91b5af04e8c5fbb06c2becc0f19d).